lib/policy_machine/policy_element.rb

```diff
@@ -249,6 +262,22 @@ def self.all(pm_storage_adapter, options = {})
       
     end
 
+    def to_s
+      unique_identifier
+    end
+
+    def operation
```

Comment:
They all have spec coverage, I feel like they're too simple to have their own specs.